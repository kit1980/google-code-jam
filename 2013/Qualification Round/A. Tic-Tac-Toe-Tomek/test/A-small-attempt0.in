10
OXXX
XOXO
TOXO
OXOX

XOOO
OXOO
XXOX
XOXX

XXTX
.XOO
.X..
OXOO

XX.O
..O.
.OX.
O..X

OOXO
XOXX
OOXT
...X

OXXX
O...
O...
T...

OOXO
XOXO
XXXO
OTXX

XXXX
XX.O
OXOO
OOXO

OOTX
OXOX
OXXX
O.XO

....
....
....
....

