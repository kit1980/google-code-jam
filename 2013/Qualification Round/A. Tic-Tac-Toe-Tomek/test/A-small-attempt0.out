Case #1: Draw
Case #2: Draw
Case #3: X won
Case #4: O won
Case #5: Game has not completed
Case #6: O won
Case #7: X won
Case #8: X won
Case #9: O won
Case #10: Game has not completed
